skiplist/src/key.rs

```diff
@@ -11,14 +12,28 @@ pub struct FixedLengthSuffixComparitor {
 }
 
 impl FixedLengthSuffixComparitor {
-    pub fn new(len: usize) -> FixedLengthSuffixComparitor {
+    pub const fn new(len: usize) -> FixedLengthSuffixComparitor {
         FixedLengthSuffixComparitor { len }
     }
 }
 
 impl KeyComparitor for FixedLengthSuffixComparitor {
```

Comment:
typo? Should `KeyComparitor`  be `KeyComparator`?

Reply:
It seems so, I'll fix.